pilot/pkg/bootstrap/monitoring.go

```diff
@@ -49,10 +49,17 @@ var (
 		"pilot_info",
 		"Pilot version and build information.",
 	)
+
+	connectionTotal = monitoring.NewGauge(
```

Comment:
duplicate with 

https://github.com/istio/istio/blob/3e427fc8a90a3e220d4556d7e92205d80916cbb8/pilot/pkg/xds/monitoring.go#L75

Reply:
I think their meanings are different. xdsClients represents how many xds streams. connectionTotal represents the number of grpc sessions.

What do you think?